datalad/resources/website/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Datalad Repository</title>
+        <link rel="shortcut icon" type="image/x-icon" href=".git/datalad/web/assets/images/favicon-16x16.png" />
+        <link rel="stylesheet" type="text/css" href=".git/datalad/web/assets/css/jquery.dataTables.css">
+        <style type="text/css" class="init">
```

Comment:
I'm guessing this allows datatable to override its css with this custom one in case of conflict or some such, not sure though